internal/settings/providers/allinkl/provider.go

```diff
@@ -0,0 +1,172 @@
+package allinkl
```

Comment:
Please add documentation in README.md and add a file for allinkl in the docs directory, thanks!!

Reply:
added in commit [[d83e297](https://github.com/qdm12/ddns-updater/pull/309/commits/d83e297632b0d491d06dbb0c576cd039600582f8)](https://github.com/qdm12/ddns-updater/pull/309/commits/d83e297632b0d491d06dbb0c576cd039600582f8)